wlm_server/pid/dac/dac8734.py

```diff
@@ -0,0 +1,78 @@
+"""Module for DAC8734 controlled via an ArtyS7 FPGA."""
+
+import serial
```

Comment:
Although `serial` is included in the `pylablib`'s dependency, how about adding `pyserial` in `requirements.txt`?